### ISSUE TITLE
Deepest cannot replicate a professional layout

### ISSUE THREAD

**Issue by Ewgeniy77** [Jack000/Deepnest#170]
Hello dear developers. I have a layout made in a professional program, the size is 4026x1380 mm. There are 52 details. I created a rectangle of the same size and marked it as a page. I sent the details to the "Start nest" layout. For two days with different installations, I could not get the same result, the maximum Deepest was able to accommodate 43 parts out of 52. Can you advise something in this situation? Thank you.

![2023-10-09_16-40-39](https://github.com/Jack000/Deepnest/assets/97045777/39367718-79af-42ad-a159-919f8ea31930)
![2023-10-10_09-11-46](https://github.com/Jack000/Deepnest/assets/97045777/73b5b40b-4dd4-4d27-875f-8461073b0bdc)
![2023-10-10_16-32-05](https://github.com/Jack000/Deepnest/assets/97045777/08194aad-9b21-4110-bd13-47d5aef2c59e)
![2023-10-10_16-34-24](https://github.com/Jack000/Deepnest/assets/97045777/1ec70f99-a55b-40ec-8552-0b6421ca491d)
![2023-10-10_16-34-45](https://github.com/Jack000/Deepnest/assets/97045777/58b1fbf4-0e1c-48f2-9e95-bc0d80bad023)

**Comment by Ewgeniy77** [comment]
![2023-10-10_16-33-29](https://github.com/Jack000/Deepnest/assets/97045777/18e1f33f-0159-44d1-bcfb-ae6ac2f09fc3)

**Comment by radevgit** [comment]
Let me ask you, why do you believe that two different products with completely different algorithms will produce the same quality results? The problem is NP complete (meaning very hard) and the produced results are usually not optimal, but "the best effort for given algorithm". There is no proof of optimality or any guarantee of the quality of the result. My guess is, that the "professional program" is using large set of placement heuristics and proprietary algos. The deepnest use some sort of genetic algo. See the references here: [https://github.com/Jack000/SVGnest](https://github.com/Jack000/SVGnest)

**Comment by Ewgeniy77** [comment]
Thank you for your reply. I did not write that I am not satisfied with the program. I asked if it was possible to make adjustments for better placement to get as close to the sample as possible. I see in the title of the program it says "industrial nesting", so I thought it might be possible.

**Comment by chetan-SMRI** [comment]
> Thank you for your reply. I did not write that I am not satisfied with the program. I asked if it was possible to make adjustments for better placement to get as close to the sample as possible. I see in the title of the program it says "industrial nesting", so I thought it might be possible.

you are right, i also think that this doesnt produce the most accurate results possible, other paid tools use different algos but produce industry standard result
